src/styles/ParentStyles/index.less

```diff
@@ -142,8 +143,77 @@
   margin: 5%;
 }
 
+//****Parent-Progress styles****
+.progress-bars {
+  font-size: 50px;
+  padding: 0% 10%;
+  display:flex;
+  flex-direction: row;
+}
+
+.progress-bar {
+  font-size: 30px;
+  padding: 10%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  size: large;
+}
+
+//****Parent-Navbar styles****
+.parent-navbar {
+  display:flex;
+  flex-direction: row;
+  justify-content: space-between;
+  text-align: center;
+  align-items: center;
+  width: 80%;
+  border: none;
+}
+
+.dropStudent {
+  padding: 12% 30%;
+  background-color: white;
+}
+
+//****Parent-Tasks styles****
+.navDrop {
+  display:flex;
+  flex-direction: row;
+  width: 100%;
+  align-items: center;
+}
+
+.classCard {
+  margin: 5%;
+}
+
+//****Parent-Resources styles****
+
+.cardContainer {
+  display: flex;
+  flex-direction: row;
```

Comment:
I believe flex-direction is row by default, so you might not need to explicitly write it. Same with flex-wrap: nowrap.